<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="suma">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>SUMA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="division">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>190</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>DIVISIÃ“N</string>
    </property>
   </widget>
   <widget class="QPushButton" name="producto">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>PRODUCTO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ROG Fonts</family>
     </font>
    </property>
    <property name="text">
     <string>primer operando</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ROG Fonts</family>
     </font>
    </property>
    <property name="text">
     <string>segundo operando</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ROG Fonts</family>
     </font>
    </property>
    <property name="text">
     <string>RESULTADO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ROG Fonts</family>
     </font>
    </property>
    <property name="text">
     <string>DECIMAL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ROG Fonts</family>
     </font>
    </property>
    <property name="text">
     <string>HEXADECIMAL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ROG Fonts</family>
     </font>
    </property>
    <property name="text">
     <string>IEEE-754</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="op1Decimal">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="op1Hexa">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="op1Bin">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="op2Decimal">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="op2Bin">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="op2Hexa">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="resultadoDecimal">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="resultadoBin">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>130</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="resultadoHexa">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>90</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
